bootstrap/pkg/kfapp/gcp/gcp.go

```diff
@@ -78,6 +78,7 @@ const (
 
 // Gcp implements KfApp Interface
 // It includes the KsApp along with additional Gcp types
+// TODO(jlewi): Why doesn't Gcp store GcpArgs as opposed to duplicating the options?
```

Comment:
not sure why this was done this way
should be
type Gcp struct {
	kfdefs.KfDef
        GcpArgs
}